src/h2olog/h2olog.h

```diff
@@ -30,10 +30,22 @@
 extern "C" {
 #include <time.h>
 }
-#include <bcc/BPF.h>
 
 class h2o_tracer
 {
+  public:
+    class usdt
+    {
+      public:
+        const std::string provider;
+        const std::string name;
+        const std::string probe_func;
+        usdt(const std::string &provider, const std::string &name, const std::string &probe_func)
+            : provider(provider), name(name), probe_func(probe_func)
+        {
+        }
+    };
```

Comment:
Would you mind elaborating the reason behind the introduction of this class?

Reply:
That's because `ebpf::USDT` doesn't have getters for the fields `provider` and `name` as of the version we use in prod, in order to filter USDTs by ther fully-qualified names.

Those accessors are [defined in the upstream BCC](https://github.com/iovisor/bcc/pull/2870), though.